README.md

```diff
@@ -86,13 +86,14 @@ need them.
 While Rustls itself is platform independent, it uses
 [`ring`](https://crates.io/crates/ring) for implementing the cryptography in
 TLS. As a result, rustls only runs on platforms
-supported by `ring`. At the time of writing, this means x86, x86-64, aarch64,
-armv7, powerpc64le, riscv64gc and s390x. For more information, see [the
-supported `ring` CI targets][ring-ci-targets].
+supported by `ring`. At the time of writing, this means 32-bit ARM, Aarch64 (64-bit ARM),
+x86, x86-64, LoongArch64, 32-bit & 64-bit Little Endian MIPS, 32-bit PowerPC (Big Endian),
+64-bit PowerPC (Big and Little Endian), 64-bit RISC-V, s390x, and WebAssembly.
```

Comment:
Actually, I think rustls doesn't support WebAssembly for reasons unrelated to the crypto?

Reply:
>  I think rustls doesn't support WebAssembly for reasons unrelated to the crypto?

Couldn't the same be true for other targets that aren't being exercised in our own CI? In general this makes me wary to use `*ring*'`s platform support as the basis of what we advertise Rustls supporting. 

In either case I've removed WebAssembly from the list (I can't build tip of main w/ `cargo build --target wasm32-unknown-unknown` despite https://github.com/rustls/rustls/issues/808 mentioning it worked at some point).

Reply:
> Couldn't the same be true for other targets that aren't being exercised in our own CI?

Maybe? It's also different with 0.17 because *ring* is now much less of a factor in restricting rustls platform support. I sort of expect WebAssembly to be the outlier case though.

Reply:
Thanks for pointing this out. *ring* 0.17.3 didn't enable Curve25519 for wasm32. *ring* 0.17.4-alpha.1 adds it and also expands the wasm32 testing to be the same as other architectures. *ring* 0.17.4 will be released in the next day or two.